### ISSUE TITLE
when I train the model ,I meet the error.

### ISSUE THREAD

**Issue by 10183308** [unsky/FPN#7]

**Comment by 10183308** [comment]
I1104 17:38:28.131677 19507 net.cpp:816] Ignoring source layer fc8
I1104 17:38:28.131716 19507 net.cpp:816] Ignoring source layer prob
Solving...
/disk03/share/FPN/tools/../lib/rpn/proposal_target_layer.py:186: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  bbox_targets[ind, start:end] = bbox_target_data[ind, 1:]
/disk03/share/FPN/tools/../lib/rpn/proposal_target_layer.py:187: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  bbox_inside_weights[ind, start:end] = cfg.TRAIN.BBOX_INSIDE_WEIGHTS
I1104 17:38:44.954255 19507 solver.cpp:229] Iteration 0, loss = 20.1489
I1104 17:38:44.954301 19507 solver.cpp:245]     Train net output #0: loss_bbox = 0.0518702 (* 1 = 0.0518702 loss)
I1104 17:38:44.954321 19507 solver.cpp:245]     Train net output #1: loss_cls = 3.63299 (* 1 = 3.63299 loss)
I1104 17:38:44.954329 19507 solver.cpp:245]     Train net output #2: rpn_cls_loss/p2 = 2.50108 (* 1 = 2.50108 loss)
I1104 17:38:44.954344 19507 solver.cpp:245]     Train net output #3: rpn_cls_loss/p3 = 2.16816 (* 1 = 2.16816 loss)
I1104 17:38:44.954352 19507 solver.cpp:245]     Train net output #4: rpn_cls_loss/p4 = 0.7777 (* 1 = 0.7777 loss)
I1104 17:38:44.954360 19507 solver.cpp:245]     Train net output #5: rpn_cls_loss/p5 = 0.678661 (* 1 = 0.678661 loss)
I1104 17:38:44.954377 19507 solver.cpp:245]     Train net output #6: rpn_loss_bbox/p2 = 8.97412 (* 1 = 8.97412 loss)
I1104 17:38:44.954399 19507 solver.cpp:245]     Train net output #7: rpn_loss_bbox/p3 = 1.36432 (* 1 = 1.36432 loss)
I1104 17:38:44.954408 19507 solver.cpp:245]     Train net output #8: rpn_loss_bbox/p4 = 0 (* 1 = 0 loss)
I1104 17:38:44.954416 19507 solver.cpp:245]     Train net output #9: rpn_loss_bbox/p5 = 0 (* 1 = 0 loss)
I1104 17:38:44.954428 19507 sgd_solver.cpp:106] Iteration 0, lr = 0.01
/disk03/share/FPN/tools/../lib/fast_rcnn/bbox_transform.py:49: RuntimeWarning: overflow encountered in exp
  pred_w = np.exp(dw) * widths[:, np.newaxis]
/disk03/share/FPN/tools/../lib/fast_rcnn/bbox_transform.py:49: RuntimeWarning: overflow encountered in multiply
  pred_w = np.exp(dw) * widths[:, np.newaxis]
/disk03/share/FPN/tools/../lib/fast_rcnn/bbox_transform.py:50: RuntimeWarning: overflow encountered in exp
  pred_h = np.exp(dh) * heights[:, np.newaxis]
/disk03/share/FPN/tools/../lib/fast_rcnn/bbox_transform.py:50: RuntimeWarning: overflow encountered in multiply
  pred_h = np.exp(dh) * heights[:, np.newaxis]
/disk03/share/FPN/tools/../lib/rpn/proposal_layer.py:183: RuntimeWarning: invalid value encountered in greater_equal
  keep = np.where((ws >= min_size) & (hs >= min_size))[0]
Floating point exception (core dumped)


Could you  help me?

**Comment by unsky** [comment]
@10183308 the reason of RuntimeWarning is occurred in the early stage because of the random state of the network , in generally it will disapper in two iter..  you can use more little learning rate

**Comment by 10183308** [comment]
@unsky  thank you ,I hava solve the problem.thank you very much

**Comment by chl916185** [comment]
I1219 12:32:56.636162  6664 sgd_solver.cpp:106] Iteration 100, lr = 0.0002
I1219 12:33:15.646114  6664 solver.cpp:229] Iteration 120, loss = 0.718101
I1219 12:33:15.646142  6664 solver.cpp:245]     Train net output #0: loss_bbox = 0.121351 (* 1 = 0.121351 loss)
I1219 12:33:15.646145  6664 solver.cpp:245]     Train net output #1: loss_cls = 0.366882 (* 1 = 0.366882 loss)
I1219 12:33:15.646149  6664 solver.cpp:245]     Train net output #2: rpn_cls_loss/p2 = 0.00236949 (* 1 = 0.00236949 loss)
I1219 12:33:15.646152  6664 solver.cpp:245]     Train net output #3: rpn_cls_loss/p3 = 0.0178403 (* 1 = 0.0178403 loss)
I1219 12:33:15.646155  6664 solver.cpp:245]     Train net output #4: rpn_cls_loss/p4 = 0.0363708 (* 1 = 0.0363708 loss)
I1219 12:33:15.646159  6664 solver.cpp:245]     Train net output #5: rpn_cls_loss/p5 = 0.224817 (* 1 = 0.224817 loss)
I1219 12:33:15.646162  6664 solver.cpp:245]     Train net output #6: rpn_loss_bbox/p2 = 0 (* 1 = 0 loss)
I1219 12:33:15.646165  6664 solver.cpp:245]     Train net output #7: rpn_loss_bbox/p3 = 0 (* 1 = 0 loss)
I1219 12:33:15.646168  6664 solver.cpp:245]     Train net output #8: rpn_loss_bbox/p4 = 0 (* 1 = 0 loss)
I1219 12:33:15.646172  6664 solver.cpp:245]     Train net output #9: rpn_loss_bbox/p5 = 0.0172386 (* 1 = 0.0172386 loss)
I1219 12:33:15.646175  6664 sgd_solver.cpp:106] Iteration 120, lr = 0.0002

loss=0？I think that's wrong，how to solve it? The data set is voc2007
@10183308 @unsky

**Comment by unsky** [comment]
@chl916185  i have update codes
